src/condor/submit.py

```diff
@@ -43,7 +43,8 @@ def main(args):
     elif args.site == "ucsd":
         t2_local_prefix = "/ceph/cms/"
         t2_prefix = "root://redirector.t2.ucsd.edu:1095"
-        proxy = "/home/users/rkansal/x509up_u31735"
+        proxy = "/home/users/annava/projects/HHbbVV/test"  # edited this to be this for Andres: grid-proxy-init -debug -verify
```

Comment:
Instead of changing this directly, can you change this to be something like:

```
username = os.environ["USER"]
if username == "rkansal":
    proxy = "/home/users/rkansal/x509up_u31735"
elif username == "annava":
    proxy = "/home/users/annava/projects/HHbbVV/test"
```

Reply:
That is better thanks!